filtered_topic_name: /location
publish_visualization: true
use_external_carstate: false
length:
  lidarToIMUDist: 1.87
  frontToIMUdistanceX: 0.0
  frontToIMUdistanceY: 0.0
  frontToIMUdistanceZ: 0.0
  rearToIMUdistanceX: 0.0
  rearToIMUdistanceY: 0.0
  rearToIMUdistanceZ: 0.0
  
